src/Language/Fixpoint/Smt/Theories.hs

```diff
@@ -63,24 +67,17 @@ import Language.Fixpoint.Utils.Builder
 -- | Theory Symbols ------------------------------------------------------------
 --------------------------------------------------------------------------------
 
+-- TODO drop all of Set and Map symbols when Map is handled through arrays
```

Comment:
is this still TODO?

Reply:
Yes, as currently the Map theory is still done via monomorphizing and it has operations for converting to Sets, we can't yet remove the SMT-interacting parts for Maps+Sets. Once Maps are also converted, we can drop this code and only leave the Array theory to be written to SMT. These TODOs serve as a reminder of this.